charts/cmc-claim-store/requirements.yaml

```diff
@@ -0,0 +1,11 @@
+dependencies:
+  - name: postgresql
```

Comment:
We happen to be maintaining postgres and java dependencies with different version than what we're using within claim-store?

Reply:
this is the helm chart version - i will look if we can pin postgres to same version as prod - i think it will be a case of defining the specific docker image in values.yaml 👍